### ISSUE TITLE
Problem of Evaluation

### ISSUE THREAD

**Issue by ZhiyeTang** [YihangChen-ee/HAC#11]
Firstly, thanks for your efforts in releasing your code. However, I found something strange in `train.py`. In line 669, the script tried to recover the latest iteration of `GaussianModel`, and evaluated its performance. Are the evaluated results consistent with what you reported in your Arxiv paper? It seems these results are based on the parameters without encoding and decoding.

Furthermore, in function `generate_neural_gaussians`, if the `GaussianModel` is in a decoded version, the neural gaussians will be directly generated by `pc._anchor_feat`, which are decoded from a bitstream encoded from `pc._anchor_feat`. In other words, in the decoded version, it seems the used features are not interpolated by grid_encoder but the original `pc._anchor_feat` instead.

**Comment by ZhiyeTang** [comment]
Thank you for your reply. I did ignore something important previously.

As to the second issue I raised, I noticed that in `generate_neural_gaussians`, the `pc._anchor_feat` will still be used to predict the corresponding `scale_rot` and `color`. That is, during rendering, the `feat_context` from `grid_encoder` is just context, assisting `mlp`s to predict the attributes of neural gaussians. Am I understanding correctly?
